python-lib/tc_etl_lib/tc_etl_lib/iota.py

```diff
@@ -102,7 +102,7 @@ def send_http(self,
             read=self.post_retry_connect,
             backoff_factor=self.post_retry_backoff_factor,
             status_forcelist=(429, 500, 502, 503, 504),
-            method_whitelist=('HEAD', 'GET', 'OPTIONS', 'POST')
+            allowed_methods=('HEAD', 'GET', 'OPTIONS', 'POST')
```

Comment:
Parece ser que este cambio se incluyó en url3 1.26.0 (https://github.com/urllib3/urllib3/blob/main/CHANGES.rst#1260-2020-11-10). Pero no podemos usar 1.26.0 en Python 3.8

```
The conflict is caused by:
    tc-etl-lib 0.13.0.post0 depends on urllib3==1.26.0; python_version < "3.12"
```

No va a quedar más remedio que usar código condicional... salvo que alguien tenga otra idea mejor :(

Reply:
¿Tal vez el antiguo método funcione en 3.8?

```suggestion
            method_whitelist=('HEAD', 'GET', 'OPTIONS', 'POST')
```

Probemos a ver...

Reply:
Parece que ha funcionado